plugins/modules/clusters.py

```diff
@@ -108,9 +113,30 @@ def run_module():
             module.fail_json(msg=f"Error deleting cluster_id: {module.params.get('cluster_id')}", **result)
         
         result = dict(changed=True, clusters=[])
+    # Retrieve cluster
+    elif module.params.get('action') == "get":
 
-    module.exit_json(**result)
+        list_params = {}
+        for k in QUERY_PARAMS_LIST:
```

Comment:
For this endpont `/v2/clusters/{cluster_id}` the params supported are different https://api.openshift.com/?urls.primaryName=assisted-service%20service#/installer/v2GetCluster I think it would need to considerate not only one query param (exclude-hosts) but two headers as well.

Reply:
I think you can omit the list_param definition because is not used in here, unless you want to add support to `exclude-hosts`.